### DISCUSSION TITLE
Noob question: DSO-5200A support?

### DISCUSSION THREAD

**Discussion by LuchkaOnTheMoon** [OpenHantek/OpenHantek6022#415] [Q&A]
Good morning people,

as per title, a very noob question: as OpenHantek6022 is a fork of OpenHantek project (which build is unfortunately failing now, sigh), should I expect that this fork is still supporting DSO-5200A (and maybe other DSO-2xxx scopes) or not?

Asking this since I'm looking for an entry level USB scope, and Hantek DSO-2150/2250/etc. have a better analog BW than 6022BE.

A side question: does OpenHantek6022 support acquiring both analog and logic signals of 6022BL at the same time?

Thank you very much in advance for your patience.

**Comment by LuchkaOnTheMoon** [comment]
A DSO-5200A is on the road to my lab, I'll test by myself then I'll leave a comment to support others.

**Comment by Ho-Ro** [comment]
> should I expect that this fork is still supporting DSO-5200A (and maybe other DSO-2xxx scopes) or not?

Nope, OH6022 supports only the 6022 and compatible scopes (based on Cypress EzUSB chip) advertised as 2 channel 48 Megasamples/second, even if it is useful only up to 24 MS/s.

You could check my branch [last_legacy_support](https://github.com/OpenHantek/OpenHantek6022/tree/last_legacy_support).
The README says:
>DSO2xxx Series and DSO52xx Series support is still in but untested due to missing devices.

**Reply by LuchkaOnTheMoon** [reply]
Thank you very much, I'm going to check your branch as soon as I receive the scope.

**Reply by LuchkaOnTheMoon** [reply]
Okay, I checked your branch for "legacy devices" support, in the README I also found this link to untested Windows binaries:

https://ci.appveyor.com/project/Ho-Ro/openhantek6022/build/artifacts

but I cannot figure out how to download last successful build.
When going back to history section, I see entries with artifacts coming from Visual Studio, but no file attached? What?

**Reply by Ho-Ro** [reply]
IF you click on artifacts you will see the message that artifacts older than 1 month will be deleted so you have to build them on your own.

**Reply by LuchkaOnTheMoon** [reply]
Never understood why all GitHub projects are not including build folder in their commits. Not a complain against you, just a fact.

I'll attempt at making my own build as soon as I have some spare time to invest, thanks for the clarification about auto-deletion message.

**Reply by LuchkaOnTheMoon** [reply]
@Ho-Ro not familiar with Qt Creator, build instructions for MS Win are saying: "Open the project in QtCreator, Compile the software" but there's no .pro file in such repository. Cmake?

**Comment by Ho-Ro** [comment]
Just open CMake.txt as the project file, QtCreator knows CMake syntax.

> Never understood why all GitHub projects are not including build folder in their commits.

For Linux it's easy:

    mkdir build
    cd build
    cmake ..
    make

I do not use Win or Mac so I cannot provide better support. Building for these platforms is done via GitHub runner; this was difficult enough for me to develop working solutions as a service.
